### PR TITLE
Transition transform instead of layout

### LINKED ISSUE #101: IE and Edge
(Accidentally posted in the v0.5 paper-slider so let's try this again)

When clicking a paper-slider in IE or Edge browsers, the resize animation for the thumb originates from a corner of the thumb, instead of the center as it should. This behavior can be seen by opening the demo for paper-slider in Edge (https://elements.polymer-project.org/elements/paper-slider?view=demo:demo/index.html&active=paper-slider). Note: the thumb returns to proper position after releasing the mouse button.

### PR DESCRIPTION
**Pull request by keanulee**
Fixes #101, #103 (make clicked knob smaller)

### CODE REVIEW

File: paper-slider.html [hunk 1]
```diff
@@ -198,24 +198,22 @@
...
         height: calc(100% - 20px);
-        border-radius: 50%;
         background-color: var(--paper-slider-knob-color, --google-blue-700);
+        border: 1px solid var(--paper-slider-knob-color, --google-blue-700);
+        border-radius: 50%;
 
         -moz-box-sizing: border-box;
         box-sizing: border-box;
 
-        transition-property: margin, height, width, background-color, border;
```
Comment (blasten) [inline paper-slider.html:207]: wow, didn't realize this element was causing layout. That might explain https://github.com/PolymerElements/paper-slider/pull/87

**Comment by keanulee** [comment]
It still does layout on certain things (e.g. dragging to the "start" state still transitions `background-color` and `border`). I now transition from `border: 2px` to `1px` when expanding to maintain a similarly-sized border after scale, but at least that only happens at the end (which is better than layout everywhere).

File: paper-slider.html (hunk 1 above)
Comment (keanulee) [inline paper-slider.html:207]: It still does layout on certain things (e.g. dragging to the "start" state still transitions `background-color` and `border`). I now transition from `border: 2px` to `1px` when expanding to maintain a similarly-sized border after scale, but at least that only happens at the end (which is better than layout everywhere).

**Comment by blasten** [comment]
lgtm. Thanks!

**Pull request merged**